dashboard/utils.py

```diff
@@ -277,6 +282,40 @@ def count_courses_passed(self):
                 passed_courses.add(self.edx_key_course_map[course_id])
         return len(passed_courses)
 
+    def get_pearson_exam_status(self):
+        """
+        Get the pearson exam status for the user / program combo
+        """
+        if self.program.exam_series_code is None:
+            return ""
+        else:
+            user = self.user
+            exam_profile = ExamProfile.objects.filter(profile=user.profile)
```

Comment:
To make this a little cleaner I would probably do:

```python
try:
    exam_profile = ExamProfile.objects.get(profile=user_profile)
except ExamProfile.DoesNotExist:
    return ExamProfile.PROFILE_ABSENT
```

Then you can ditch lines 295-296 and all the `[0]` shenanigans.